train_network.py

```diff
@@ -0,0 +1,179 @@
+import tensorflow as tf
+import numpy as np
+from accuracy import *
+from DataReader import *
+from DRN import *
+import os
+from dice import *
+
+tf.reset_default_graph()
+directory = os.getcwd()
+
+train_file = os.path.join(directory, 'data', 'train.txt')
+test_file =  os.path.join(directory, 'data', 'val.txt')
+logs = os.path.join(directory, 'logs')
+trainloss = os.path.join(logs, 'train_loss.txt')
+
+
+if os.path.isdir(logs) == False:
+  os.makedirs(logs)
+
+# choose network, can be either DRN18 or DRN26
+network = 'DRN26'
```

Comment:
parameters are better off for users to set via `argparse` instead of hardcoding at the top. This is common for scripts but is generally bad practice for writing re-usable/productionizable code or when collaborating on a codebase as a team.